src/screens/MainScreen/index.js

```diff
@@ -51,7 +81,19 @@ const Main = ({ navigation }) => {
           location={currentLocation}
           markerKey={0}
           showCircle
+          draggable
         />
+        {targetsList &&
+          targetsList.map(target => (
```

Comment:
```suggestion
          targetsList.map({id, topic, lat, lng, radius} => (
```

and then you can use those values directly